api/api_test.go

```diff
@@ -5,10 +5,14 @@ import (
 	"fmt"
 	"net/http"
 	"net/http/httptest"
+	"strings"
 	"testing"
 	"time"
 
+	mocks "github.com/hashicorp/consul-terraform-sync/mocks/driver"
+
```

Comment:
Not sure if this is intentional to have this import grouped by itself

Reply:
Not intentional, I have a future set of changes that modifies this files as well. I've updated that set to include regrouping this import with the other cts imports. Thanks!

Reply:
Note: following up, here's the commit to regroup the import https://github.com/hashicorp/consul-terraform-sync/pull/196/commits/4b040ed32228aff0fdadc6049f46939afcb16e5e, part of PR-196